src/mlpack/methods/softmax_regression/softmax_regression_function.cpp

```diff
@@ -205,3 +205,16 @@ void SoftmaxRegressionFunction::Gradient(const arma::mat& parameters,
                lambda * parameters;
   }
 }
+
+void SoftmaxRegressionFunction::FeatureGradient(const arma::mat& parameters,
+    const size_t j,
```

Comment:
You could also tab this to be under `c` in the above line, but that's really minor